### PR TITLE
Space endpoint

### PR DESCRIPTION
**Pull request by Jonathan6**
Added a page for viewing Space pages based on their UUID. Created mini Idea handlebars partial to display on the page.

### CODE REVIEW

**Comment by ac524** [comment]
Always a good idea to checkout other work people have in progress before starting yours :) If you see another member has a big set of changes pending to merge in like Kim did, then it's not a bad idea to consider branching off their code instead of `main` and make pull requests into their branch. That's what I ended up doing with all main changes yesterday, so I could avoid merge conflicts as best I could.

**Comment by kimberlym4488** [comment]
Totally my bad. Once I started on the homepage functionality I had to add in the space routes to get the homepage buttons to route properly. Should have opened a new branch for just that, separate from login/logout. Good lesson learned! :)

**Review by kimberlym4488 — APPROVED** [review]
Awesome work! I like the idea partial :)

**Review by jmichaelbrown8 — APPROVED** [review]
Looks great! Thanks

File: views/partials/ideaMini.handlebars [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{{#each ideas}}
```
Comment (ac524) [inline views/partials/ideaMini.handlebars:1]: With the update the how we fetch the data in the route, this should be updated to `{{#each space.ideas}}` or something similar. (Look at the actual `space` data structure by console logging it in the route.

**Review by ac524 — CHANGES_REQUESTED** [review]
The @Jonathan6 @kimberlym4488 added some recommendations on data loading for the space view.

**Comment by ac524** [comment]
Got the updates I suggested done, and updated styling for the idea items on the space page. Think this is ready for merging! If someone can give a final review :)

**Review by kimberlym4488 — APPROVED** [review]
Thanks for all the updates!

**Pull request merged**